src/main/java/com/esri/samples/na/closest_facility_static/ClosestFacilityStaticSample.java

```diff
@@ -0,0 +1,344 @@
+            ClosestFacilityParameters closestFacilityParameters = closestFacilityTask.createDefaultParametersAsync().get();
+            closestFacilityParameters.setFacilities(facilitiesList);
+            closestFacilityParameters.setIncidents(incidentsList);
+
+            // solve closest facilities
+            try {
+              // use the task to solve for the closest facility
+              ListenableFuture<ClosestFacilityResult> closestFacilityTaskResult = closestFacilityTask.solveClosestFacilityAsync(closestFacilityParameters);
+              closestFacilityTaskResult.addDoneListener(() -> {
+                try {
+                  ClosestFacilityResult closestFacilityResult = closestFacilityTaskResult.get();
+
+                  // find the closest facility for each incident
+                  for (int indexOfIncident = 0; indexOfIncident < incidentsList.size(); indexOfIncident++) {
+
+                    // get the index of the closest facility to incident. (i) is the index of the incident, [0] is the index of the closest facility.
+                    Integer closestFacilityIndex = closestFacilityResult.getRankedFacilityIndexes(indexOfIncident).get(0);
+
+                    // get the route to the closest facility.
+                    ClosestFacilityRoute closestFacilityRoute = closestFacilityResult.getRoute(closestFacilityIndex, indexOfIncident);
+
+                    // display the route on the graphics overlay
+                    mapView.getGraphicsOverlays().get(0).getGraphics().add(new Graphic(closestFacilityRoute.getRouteGeometry(), simpleLineSymbol));
+
+                    // disable the solve button
+                    solveRoutesButton.setDisable(true);
+
+                    // enable the reset button
+                    resetButton.setDisable(false);
+                  }
+
+                } catch (ExecutionException | InterruptedException e) {
+                  displayMessage("Error getting the ClosestFacilityTask result", e.getMessage());
+                }
+              });
+
+            } catch (Exception e) {
+              displayMessage("Error solving the ClosestFacilityTask", e.getMessage());
+            }
+
+          } catch (InterruptedException | ExecutionException e) {
+            displayMessage("Error getting default route parameters", e.getMessage());
+          }
+
+        } else {
+          displayMessage("Error loading route task", closestFacilityTask.getLoadError().getMessage());
+        }
+      });
+
+    } catch (Exception e) {
+      e.printStackTrace();
+    }
+  }
+
+  /**
+   * Creates a PictureMarkerSymbol from a URI and sizes it appropriately
+   *
+   * @param uri the URI of the picture to be used for the symbol
+   */
+  private PictureMarkerSymbol createSymbol(String uri) {
+    PictureMarkerSymbol symbol = new PictureMarkerSymbol(uri);
+    symbol.setHeight(30);
+    symbol.setWidth(30);
+    return symbol;
+  }
+
+  private void resetRoutes() {
```

Comment:
this method is only used once and is fairly short so doesn't need to be separated out. Move the code out of this method directly into where it is used in the `start` method.

N.B. if we were keeping this in, it would need a Javadoc comment. Just a note for future apps :)